stock_measuring_device/data/uom.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="product_uom_mm" model="uom.uom">
+        <field name="category_id" ref="uom.uom_categ_length" />
+        <field name="name">mm</field>
+        <field name="factor" eval="1000" />
+        <field name="uom_type">smaller</field>
+    </record>
+</odoo>
```

Comment:
We should drop this in favor of the core one:

```
https://github.com/odoo/odoo/blob/ae0b3d93481baff67d3cac9857f4e073911d9840/addons/uom/data/uom_data.xml#L61-L66
```


- https://github.com/odoo/odoo/pull/64735